src/solidity/types/util.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  extractValue: function (slotValue, storageBytes, location) {
```

Comment:
`extractHexByteSlice`? and perhaps `storageBytes` should be `byteLength`, and `location` should be replaced by `offsetFromLSB`